Cargo.toml

```diff
@@ -103,22 +100,13 @@ max_level_off = ["log/max_level_off"]
 alloc = ["rustix/alloc", "smallvec"]
 
 # Enable support for threads.
-thread = ["rustix/thread", "rustix/mm", "param", "rustix/process", "rustix/runtime", "rustix-futex-sync"]
+thread = ["rustix/thread", "rustix/mm", "rustix/param", "rustix/process", "rustix/runtime", "rustix-futex-sync"]
 
 # Enable support for signal handlers.
 signal = ["rustix/runtime"]
 
-# Enable support for `rustix::param`.
-param = ["rustix/param"]
-
 # Enable support for ELF `.init_array` and `.fini_array`.
-init-fini-arrays = ["init-array", "fini-array"]
-
-# Enable support for ELF `.init_array`.
-init-array = []
-
-# Enable support for ELF `.fini_array`.
-fini-array = []
```

Comment:
c-scape enables `.fini_array` support too already.

Reply:
c-scape does, but only because `.fini_array` is a well-known feature of ELF and user code could in theory depending on it. But it's nice if applications using origin directly can disable it if they don't need it.

Reply:
Alright, dropped the commit.